proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -218,6 +218,13 @@ message GetLatestCuratedPluginRequest {
 }
 
 message GetLatestCuratedPluginResponse {
```

Comment:
I guess this and L116 are not true anymore then?

```suggestion
message GetCuratedPluginResponse {
```

Although this'd be a breaking change. So may be just a clarification point and not an actual suggestion.

Reply:
If you omit the version in the request, then it does return the latest curated plugin. And if you include the version, then it'll return the latest curated plugin by revision.

We might want to add revision as an optional, but that hasn't come up yet.

Reply:
Gotcha, thanks for the clarification :)